metrics/stackdriver_exporter_test.go

```diff
@@ -185,15 +185,11 @@ func TestGetMonitoredResourceFunc_UseKnativeBroker(t *testing.T) {
 		if got != testEventType {
 			t.Errorf("expected new tag: %v, got: %v", eventTypeKey, newTags)
 		}
-		got = getResourceLabelValue(metricskey.LabelEventSource, newTags)
```

Comment:
Do you want to check `LabelSourceName` instead?

Reply:
Yeah, but I'll change it to Name.

Thanks for taking a look! Will ping you tmrw for another round :)

Reply:
this label has been dropped from KnativeBroker, so it's fine not to check this